### PR TITLE
German Translation

### PR DESCRIPTION
**Pull request by Lime712**
If your pull request is **not** translation or serverlist-related, read the list of requirements below and check each box:

- [ ] I have read the [contribution guidelines](https://github.com/Anuken/Mindustry/blob/master/CONTRIBUTING.md).
- [ ] I have ensured that my code compiles, if applicable.
- [ ] I have ensured that any new features in this PR function correctly in-game, if applicable.

### CODE REVIEW

File: core/assets/bundles/bundle_de.properties [hunk 1]
```diff
@@ -965,7 +965,7 @@ ability.statusfield = {0} Statusfeld
 ability.unitspawn = {0} Fabrik
 ability.shieldregenfield = Schild-regenerations-Feld
 ability.movelightning = Bewegungsblitze
-ability.shieldarc = Lichtbogenschild
+ability.shieldarc = Bogenschild
```
Comment (Phippe) [inline core/assets/bundles/bundle_de.properties:968]: If you're really going to change it, you should also change the other translations of "arc".

Also, shouldn't it be "Schild(licht)bogen" anyway? The original word is shield arc, not arc shield.

File: core/assets/bundles/bundle_de.properties [hunk 2]
```diff
@@ -965,7 +965,7 @@ ability.statusfield = {0} Statusfeld
 ability.unitspawn = {0} Fabrik
 ability.shieldregenfield = Schild-regenerations-Feld
```
Comment (Phippe) [inline core/assets/bundles/bundle_de.properties:966]: Might as well change this to "Schildregenerationsfeld" while we're here. Or at least capitalise the middle word.

**Review by Phippe — CHANGES_REQUESTED** [review]

File: core/assets/bundles/bundle_de.properties [hunk 3]
```diff
@@ -963,9 +963,9 @@ ability.forcefield = Kraftfeld
 ability.repairfield = Heilungsfeld
 ability.statusfield = {0} Statusfeld
 ability.unitspawn = {0} Fabrik
-ability.shieldregenfield = Schild-regenerations-Feld
+ability.shieldregenfield = Schildregenerationsfeld
 ability.movelightning = Bewegungsblitze
-ability.shieldarc = Lichtbogenschild
+ability.shieldarc = Schildbogen
```
Comment (Catchears) [inline core/assets/bundles/bundle_de.properties:968]: I'd want to stay with 'Lichtbogen' for clarity and uniformity personally - I'd also prefer 'Lichtbogenschild' over 'Schildlichtbogen' as we're referring to an arc-shaped shield and not an arc that also happens to be a shield. Open to other ideas though.

Comment (Phippe) [reply core/assets/bundles/bundle_de.properties:968]: Sure, I'm fine with the old order too, it was more of an observation anyway

**Pull request merged**